app/src/main/res/values/string-not_translatable.xml

```diff
@@ -297,6 +388,8 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="char_punctation_mark_arrows" translatable="false">Punctuation mark (»«)</string>
     <string name="indent" translatable="false">Indent</string>
     <string name="deindent" translatable="false">De-indent</string>
+    <string name="indent_level" translatable="false">Indent Level</string>
+    <string name="deindent_level" translatable="false">De-indent Level</string>
```

Comment:
there are existing translations for indent and de-indent, aren't there?

Reply:
I implemented 2 different kinds of indent and unindent. They are doing very different things. Therefore different icons and different names (translations) are required.

Reply:
type `## Hello` in Markdown file with and without bigger headings enabled, and see yourself.